packages/code-editor/src/CodeEditor/testing/testUtils.tsx

```diff
@@ -24,7 +64,8 @@ function getBySelector(
   selector: CodeEditorSelectors,
   options?: { text?: string },
 ) {
-  const elements = editorView.dom.querySelectorAll(selector);
+  const view = ensureEditorView();
+  const elements = view.dom.querySelectorAll(selector);
```

Comment:
Future consideration: Might be useful to have `find...` utils that use `waitForEditorView` under the hood